lib/Test/PostgreSQL.pm

```diff
@@ -335,6 +370,7 @@ method DEMOLISH($in_global_destruction) {
     if (defined $self->pid && $self->_owner_pid == $$) {
       $self->stop
     }
+    undef $self->{_socket_dir};
```

Comment:
This _shouldn't_ be necessary, I just felt better about cleaning it up explicitly since I'm not very familiar with object destruction sequence of the library we're using. It's been a long time since I've done any development on it.